api/BUILD

```diff
@@ -164,6 +164,16 @@ api_proto_library(
     ],
 )
 
+api_proto_library(
+    name = "accesslog",
+    srcs = ["accesslog.proto"],
+    has_services = 1,
+    deps = [
+        ":base",
+        "@com_google_protobuf//:timestamp_proto",
```

Comment:
I think if you move this to https://github.com/envoyproxy/data-plane-api/blob/master/bazel/api_build_system.bzl#L31 you will fix the build issue. The idea is we centralize the WKT dependencies.